remo/domain/dataset.py

```diff
@@ -260,6 +262,7 @@ def export_annotations(
                 annotation_format=annotation_format,
                 export_coordinates=export_coordinates,
                 full_path=full_path,
+                export_tags=str(export_tags).lower()
```

Comment:
@SreeHarshaNelaturu here you can pass `export_tags` value as it is

```suggestion
                export_tags=export_tags
```

Reply:
Its a boolean value, is that okay?